src/app_engine/analytics.py

```diff
@@ -75,7 +75,8 @@ def report_event(self, event_type, room_id=None, time_ms=None,
       event[LogField.FLOW_ID] = flow_id
 
     if client_type is not None:
-      event[LogField.CLIENT_TYPE] = client_type
+      client_type_name = ClientType.Name.get(client_type, str(client_type))
```

Comment:
We can do that but it's a separate issue. The client was sending the correct data in this case.

I think we should push this change, I filed #141 to track the other issue.